examples/funloc/funloc_params.yml

```diff
@@ -98,4 +98,4 @@ report:
   whitening: {analysis: All, name: All}
   sensor: {analysis: All, name: All, times: [0.1, 0.2]}
   source: {analysis: All, name: All, inv: '%s-50-sss-meg-eeg-free-inv.fif',
-           times: [0.09, 0.4], views: lat, size: [800, 400]}
+           times: 'peaks', views: lat, size: [800, 400]}
```

Comment:
It would be good to also use `"times"` for the `sensor` category above for consistency

Reply:
Do you mean `times='peaks'` for the sensor category?  I figured showing `times` set in both modes (an array and a string) best shows the available options to a new user.  Alternatively, detailing all options for the various report parameters is something that perhaps should be added to the ever-improving Overview doc.

Reply:
I'd rather leave the detail-all-options in the docs section, the example works better for showing good, consistent option usage